Lab3/ComputerStore.WebUI/wwwroot/lib/bootstrap/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
I can't find the config file for client libraries (libman.json).
Here the info about this file: https://docs.microsoft.com/en-us/aspnet/core/client-side/libman/libman-vs?view=aspnetcore-3.1
And I see you uploaded 'wwwroot/lib' which should absent in repository because LibMan config allows you to restore all of them.
You need to check your '.gitignore' file and fix it.
Here the project sample for better understanding of what I mean:
https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/client-side/libman/samples/LibManSample